jujugui/static/gui/src/app/store/env/controller-api.js

```diff
@@ -446,7 +446,7 @@ YUI.add('juju-controller-api', function(Y) {
             owner: tags.parse(tags.USER, result['owner-tag']),
             life: result.life,
             isAlive: result.life === 'alive',
-            isAdmin: result.uuid === result['controller-uuid']
+            isAdmin: result.name === 'controller'
```

Comment:
lets change this name, the `isAdmin` doesn't make much sense now that it's a controller. `isController` is more appropriate.